tools/tune/dmic/dmic_init.m

```diff
@@ -137,6 +136,7 @@
 fir.rs = rs;
 fir.m = mfir;
 dmic_fir_export(fir, 'include');
+[~, prm.vfn] = dmic_fir_export_alsa(fir, 'include_alsa_utils', prm.vfn);
 
 end
 
```

Comment:
Please think about substituting the phrase `if length(common_mcic_list) > 0` at line 159 with the phrase `isempty(common_mcic_list)`.

Reply:
Please consider changing line 182 with` cfg = get_cic_config(prm, cfg, hw) ` `cfg = get_cic_config( cfg, hw)` as the `prm` variable is not being used.

Reply:
Take into account eliminating the superfluous` semicolon` at lines 193 and 198.

Reply:
Thanks, I did that cleanup too. I've checked that none of the exported coefficients change due to these changes, so they were safe to do.